test/ecpair.js

```diff
@@ -299,4 +299,18 @@ describe('ECPair', function () {
       }))
     })
   })
+
+  describe('fromPrivateKeyBuffer', function () {
+    var ecPair
+
+    before(function () {
+      ecPair = ECPair.makeRandom()
+    })
+
+    it('reconstructs an ECPair from the private key buffer', function () {
+      var prvKeyBuffer = ecPair.getPrivateKeyBuffer()
+      var reconstructedEcPair = ECPair.fromPrivateKeyBuffer(prvKeyBuffer)
```

Comment:
Add a test using different networks

Reply:
I'm not sure what that would accomplish. `fromPrivateKeyBuffer`'s behavior does not depend on the network.

Reply:
Ah true, nvm.